lib/Clownface.js

```diff
@@ -273,8 +273,8 @@ class Clownface {
   }
 
   /**
-   * Creates a graph pointer to nodes which link the current pointer by `predicates`
-   * @param {Term|Term[]|Clownface|Clownface[]} [predicates] one or more RDF/JS term identifying a property
+   * Creates a graph pointer to nodes that are subjects of a predicate, or of any predicates in an array
```

Comment:
Sorry for the wrong info in the chat, but it should be `object`. I'm not sure if "subjects of a predicate" is the right wording. I think it should include something about traversing/following. Maybe:

> Creates a graph pointer to the result nodes after following a predicate, or of any predicates in an array, starting from the subjects to the objects.

Reply:
Alright, what about:
> Creates a graph pointer to the result nodes after following a predicate, or after following any predicates in an array, starting from the subject(s) (current graph pointer) to the objects.

Reply:
:+1: